src/encoded/schemas/library.json

```diff
@@ -95,6 +95,11 @@
             "type": "number",
             "minimum": 0
         },
+        "antibody": {
```

Comment:
May want to add some enum or suggested enum here or you are likely to get a wide variety of values input.  In 4DN we have an antibody item and the 'title' is a calcprop that combines target and vendor.  If an Item is not needed (it likely isn't) then you may want to set up some rules for naming conventions.